views/default/input/number.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Elgg number input
+ * Displays a number input field
+ *
+ * @package Elgg
+ * @subpackage Core
+ *
+ * @uses $vars['class'] Additional CSS class
+ */
+
+$vars['class'] = (array) elgg_extract('class', $vars, []);
```

Comment:
Could you change this to elgg_extract_class(). @mrclay just cleaned up all of the input views